### ISSUE TITLE
Deprecation warning over invalid escape sequences

### ISSUE THREAD

**Issue by tirkarthi** [Rapptz/discord.py#2534]
### Summary

Deprecation warning over invalid escape sequences. Using raw string or escaping the relevant part will fix this.

### Reproduction Steps

```
./discord/ext/commands/core.py:1427: DeprecationWarning: invalid escape sequence \*
  """A :func:`check` that is added that checks if any of the checks passed
```

### Checklist

- [x] I have searched the open issues for duplicates.
- [x] I have shown the entire traceback, if possible.
- [ ] I have removed my token from display, if visible.

### System Information

master branch

**Comment by XuaTheGrate** [comment]
Can you provide proper version information and reproduction steps? I am not seeing this on latest with Python 3.7.4, discord.py 1.3.0.

**Comment by tirkarthi** [comment]
I am using Python 3.8 on Linux. I guess it's a warning on 3.7 too. Command as below to compile all Python files that shows this warning.

```
find . -iname '*.py'  | xargs -P 4 -I{} python3.8 -Wall -m py_compile {} 
./discord/ext/commands/core.py:1427: DeprecationWarning: invalid escape sequence \*
  """A :func:`check` that is added that checks if any of the checks passed
```

**Comment by Rapptz** [comment]
Closed through #2537
